public/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "public",
```

Comment:
In a normal firebase hosting deployment, the entire "public" directory would be deployed to hosting and served to the public web.  So you'd only want your output html / css files in `public`.  By putting `package.json` in here, you'd be making it available to the public web.  Given that we are rerouting most requests to the functions handler, this setup is a bit different, but I think it'd still make sense to keep our source files and package.json, etc. out of the `public` directory.  Among other things, I'm guessing the hosting deploy is going to upload it all during deploys, even though the rewrite rules will prevent directly accessing it.

So I'd move this content up to the root package.json.  In general it's a bit unusual to have multiple package.json files in a repo.  Firebase functions is a bit of a special case because it needs to install/run the code on a remote server, so it needs to be all isolated in a deployable package.  And in act-now-packages we have multiple packages we are publishing to npm so we need multiple package.json files.  But typically, for most projects, I'd just expect a single package.json at the root (at least in my experience).

Reply:
This makes a lot of sense, thanks for this explanation! 🙏 I've moved the config and `.ts` files up to the top-level